backend/typescript/graphql/index.ts

```diff
@@ -34,17 +36,22 @@ const mutation = gql`
 
 // no-auth {
 const executableSchema = makeExecutableSchema({
-  typeDefs: [query, mutation, entityType],
-  resolvers: merge(entityResolvers),
+  typeDefs: [query, mutation, entityType, simpleEntityType],
```

Comment:
Please resolve merge conflicts with release-v2 before merging, we should not have auth/no-auth tags any longer :)